### ISSUE TITLE
Error in getsRNATrinucleotidesFrequncies

### ISSUE THREAD

**Issue by ramostapiai** [BioinformaticsLabAtMUN/sRNARFTarget#1]
Hello there,

I am working with sRNATarget with ~ 200 bacterial sRNAs to find the interaction with human mRNAs. (200000 sequences). The first step is fine, but the second step getsRNATrinucleotidesFrequncies call gives me an error. Maybe you could help me.

Thanks for your help
I using a macOS 11.6
i9 processors
64 Gb RAM
1 Tb SSD
Error:
Error executing process > 'getsRNATrinucleotidesFrequncies (1)'

Caused by:
  Process `getsRNATrinucleotidesFrequncies (1)` terminated with an error exit status (137)

Command executed:

  #!/usr/bin/env python3
  from pprint import pprint
  from skbio import Sequence
  from itertools import product
  from sklearn.utils import shuffle
  import pandas as pd
  import numpy as np
  
  
  data1 = pd.read_csv('pairs_names_seqs.txt', sep='	', header=0)
  df = pd.DataFrame(data = data1)
  sequences = df.iloc[:, 2].values #third column sRNA sequences
  
  seqarr = []
  
  for item in sequences:
      number = 1
      s = Sequence(item)
executor >  local (2)
[e0/a95007] process > createAllPossiblePairs              [100%] 1 of 1 ✔
[a6/594954] process > getsRNATrinucleotidesFrequncies (1) [100%] 1 of 1, failed: 1 ✘
[-        ] process > getmRNATrinucleotidesFrequncies     -
[-        ] process > runRandomForestModel                -
[-        ] process > generateSortedResultFile            -
Error executing process > 'getsRNATrinucleotidesFrequncies (1)'

Caused by:
  Process `getsRNATrinucleotidesFrequncies (1)` terminated with an error exit status (137)

Command executed:

  #!/usr/bin/env python3
  from pprint import pprint
  from skbio import Sequence
  from itertools import product
  from sklearn.utils import shuffle
  import pandas as pd
  import numpy as np
  
  
  data1 = pd.read_csv('pairs_names_seqs.txt', sep=' (rep', header=0)
  df = pd.DataFrame(data = data1)
  sequences = df.iloc[:, 2].values #third column sRNA sequences
  
  seqarr = []
  
  for item in sequences:
      number = 1
      s = Sequence(item)
      freqs = s.kmer_frequencies(3, relative=True, overlap=True)
      seqarr.append(freqs)
      number = number + 1
  
  def all_kmer_subsets(ss=["A", "T", "G", "C"]):
      return [''.join(p) for p in product(ss, repeat=3)]
  
  kmer_combinations = all_kmer_subsets()
  
  df1 = pd.DataFrame(seqarr) #convert dicionary to dataframe
  rows = len(df1.index)
  cols = len(kmer_combinations)
  d = np.zeros(shape=(rows,cols))
  df2 = pd.DataFrame(data = d, columns=kmer_combinations)
  df3 = pd.DataFrame()
  
  for col in kmer_combinations:
      if col in df1.columns:
          df3[col] = df1[col]
      else:
          df3[col] = df2[col]
  
  df3 = df3.fillna(0) #fill empty columns with zero (replace NaN with 0)
  
  df3.to_csv('sRNA_3mer.txt', header=True, index=False, sep='	', mode='a')

Command exit status:
  137

Command output:
  (empty)

Work dir:
  /Users/ignacioramos/Documents/programs/sRNARFTarget/work/a6/59495492270cccba0cd605e8f31e45

Tip: you can try to figure out what's wrong by changing to the process work dir and showing the script file named `.command.sh`
